src/app/routes/article/getInitialData/index.ts

```diff
@@ -27,10 +26,7 @@ type Props = {
 };
 
 const transformPageData = (toggles?: Toggles) =>
-  pipe(
-    addAnalyticsCounterName,
-    augmentWithDisclaimer({ toggles, positionFromTimestamp: 0 }),
-  );
+  pipe(augmentWithDisclaimer({ toggles, positionFromTimestamp: 0 }));
```

Comment:
Is pipe still needed here?

Reply:
Good spot, it can be removed since there's only 1 function